src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.cs

```diff
@@ -273,8 +273,8 @@ public TargetFrameworkMoniker TFM
 
                     if (propCount != 1)
                     {
-                        _logger.LogError("Non-SDK projects being upgraded must have exactly one TargetFrameworkVersion property. Found {TargetFrameworkVersionCount} TargetFrameworkVerion properties.", propCount);
-                        throw new UpgradeException($"SDK projects being upgraded must have exactly one TargetFrameworkVersion property. Found {propCount} TargetFrameworkVersion properties.");
+                        _logger.LogError("Non-SDK projects being upgraded must have exactly one TargetFrameworkVersion property. Found {TargetFrameworkVersionCount} TargetFrameworkVerion properties for project {Project}.", propCount, FilePath);
```

Comment:
Opportunity for local method to reduce redundancy?

Reply:
Sure. I'll pull out the common bits.